tools/add_font.py

```diff
@@ -21,6 +50,9 @@
 flags.DEFINE_integer('min_pct_ext', 10,
                      'What percentage of subset codepoints have to be supported'
                      ' for a -ext subset.')
+flags.DEFINE_boolean('update', False,
```

Comment:
WDYT about default True or deleting the flag entirely? - I'm thinking that usually if I point add_font at an existing directory with a METADATA.pb I don't want it to destroy information.